cmd/gen-test-vectors/main.go

```diff
@@ -163,6 +169,12 @@ func GenerateTestVectors(ctx context.Context, env *integration.Env) error {
 			Resp:   e,
 		})
 
+		// Let the trusted log root fall behind by a few revisions
+		// to make consistency proofs more interesting.
+		if newslr.TreeSize%5 == 1 {
```

Comment:
nit: might be cleaner to compare with 0 -- unless you're using 1 for a reason in which case it would be nice to document it.

Reply:
Documented that we're using 1 intentionally. (There aren't more than 5 test cases at the moment, so 0 would use the empty trusted log root which doesn't require consistency proofs.)